README.md

```diff
@@ -14,7 +14,44 @@ This package is written for the Home Assistant [ha-tahoma](https://github.com/iM
 ## Installation
 
 ```bash
-pip install git+https://github.com/iMicknl/python-tahoma-api.git@master#tahoma_api
+pip install pyhoma
+```
+
+## Getting started
+
+```python
+import asyncio
+import time
+
+from tahoma.client import TahomaClient
```

Comment:
Just change this import to pyhoma, and I'm good.

Reply:
Forgot to push again..